### DISCUSSION TITLE
Different file sorting methods

### DISCUSSION THREAD

**Discussion by nitrostudios** [mikekazakov/nimble-commander#259] [Ideas]
For example:

![Screen Shot 2024-04-26 at 14 53 23](https://github.com/mikekazakov/nimble-commander/assets/157495999/e63cae24-d1af-42d0-98d3-5e57edbd6880)

**Comment by ilirium** [comment]
It would be really good to sort files and folders like it is implemented in Finder. As I [find](https://apple.stackexchange.com/questions/381772/is-there-a-way-to-change-the-order-in-which-the-finder-alphabetizes):

> The Finder’s sort order for file and directory names is based on the Unicode Collation Algorithm (Technical Standard UTS #10) defined by the Unicode Consortium.

This is kind of Natural Sorting Order.

**Comment by mikekazakov** [comment]
Yes, it would be good having an _option_ to use natural sorting order - PRs are welcome.

**Comment by mikekazakov** [comment]
I've decided to implement this feature for v1.7, more details will be in the issue #85.

**Comment by mikekazakov** [comment]
Done.
